### PR TITLE
courier-react tweaks

### PR DESCRIPTION
**Pull request by rileylnapier**
## Description

- Unread and read notification should be aligned. The unread notification below currently has an indent
- The preferences gear and the "mark all as read" cta should both be #73819B. 
- Change branding so the tabs, notification CTAs and unread indicator are all the primary CTA. The secondary/tertiary branding will be covered by our background gradient in inbox v2
- If possible we should scale the inbox down to 360px width. That's what we have in our figma.
- Notification border radius
- When should the toast notification disappear
- Can we tweak the toggle so it follows this toggle design?
- The alignment in the preference grouping looks off. can we adjust so the margin above the preference category title and below opted in is the same

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/react-toast/src/components/Toast/index.tsx [hunk 1]
```diff
@@ -34,8 +34,16 @@ export const Toast: React.FunctionComponent<
...
+  const {
+    clientKey,
+    transport,
+    dispatch,
+    brand: courierBrand,
+  } = courierContext;
 
+  const brand = config?.brand ?? courierBrand;
+
+  console.log("brand", brand);
```
Comment (tk26) [inline packages/react-toast/src/components/Toast/index.tsx:46]: remove

**Review by tk26 — DISMISSED** [review]

File: packages/react-toast/src/components/Toast/index.tsx (hunk 1 above)
Comment (suhasdeshpande) [reply packages/react-toast/src/components/Toast/index.tsx:46]:
```suggestion

```

File: packages/storybook/stories/toast/examples.stories.tsx [hunk 2]
```diff
@@ -124,6 +124,9 @@ export function WithCourierProvider({ bodyText }) {
               message:
                 "https://d33wubrfki0l68.cloudfront.net/ca2747f11cc64d0e424e27b4a804b9d981b22453/9ab46/_next/static/images/logo@2x-5d5af82635bfdd3ad24e54f9eb364097.png",
             },
+            toast: {
+              autoCloseTimer: 0,
```
Comment (suhasdeshpande) [inline packages/storybook/stories/toast/examples.stories.tsx:128]: I was not able to see inbox examples in deploy preview for some reason?

Comment (rileylnapier) [reply packages/storybook/stories/toast/examples.stories.tsx:128]: its because the preview is missing env variables so the story borks

**Review by tk26 — APPROVED** [review]

**Pull request merged**